zhazam/Modules/StartGameTimer/StartGameTimerViewController.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  StartGameTimerViewController.swift
+//  zhazam
+//
+//  Created by Abai Kalikov on 4/22/20.
+//  Copyright © 2020 crococoders. All rights reserved.
+//
+
+import UIKit
+
+final class StartGameTimerViewController: UIViewController {
+    
+    private enum Constants {
+        static let counterArray: [String] = ["3", "2", "1", ""]
+        static let lastRowDisplayIndex = 1
+        static let pickerViewRowHeight: CGFloat = 200
+        static let reusingViewFontSize: CGFloat = 140
+        static let timerTimeInterval: TimeInterval = 1.0
+    }
+    
+    @IBOutlet weak var timerPickerView: UIPickerView!
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        configurePickerView()
+    }
+    
+    private func configurePickerView() {
+        timerPickerView.dataSource = self
+        timerPickerView.delegate = self
+        timerPickerView.isUserInteractionEnabled = false
+    }
+    
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        
+        makeTimer { [weak self] in
+            guard let self = self else { return }
```

Comment:
You probably do not need to check for self here. Write me and we'll discuss it

Reply:
Ok, got it, already changed.